eos/rare-b-decays/b-to-psd-nu-nu.cc

```diff
@@ -180,7 +180,12 @@ namespace eos
             const double m_P = this->m_P(), m_P2 = m_P * m_P;
             const double m_b = model->m_b_msbar(mu);
             const double m_s = model->m_s_msbar(mu);
-            const double lambda = eos::lambda(m_B2, m_P2, q2), sqrt_lambda = std::sqrt(lambda);
+            double lambda = eos::lambda(m_B2, m_P2, q2);
```

Comment:
I am not sure that I like this. My default approach would be to keep ``lambda`` const and run
```c++
if (lambda < 0)
{
    return 0;
}
```

Reply:
Ok! I was trying to follow what we have elsewhere in EOS, but I am happy to amend that :)

Reply:
Maybe I am off. My reasoning would be that
1. the computation of the observable might not always be well defined for ``lambda = 0``;
2. the return of 0 immediately avoids unnecessary computations.

Reply:
I will change that